packages/module/patternfly-docs/content/extensions/data-view/examples/Toolbar/Toolbar.md

```diff
@@ -5,30 +5,55 @@ section: extensions
 # Sidenav secondary level section
 # should be the same for all markdown files
-id: Functionality
+id: Toolbar
+title: Data view toolbar
 # Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
 source: react
 # If you use typescript, the name of the interface to display props for
 # These are found through the sourceProps function provided in patternfly-docs.source.js
-sortValue: 3
-propComponents: ['DataViewFilters', 'DataViewTextFilter', 'DataViewCheckboxFilter']
-sourceLink: https://github.com/patternfly/react-data-view/blob/main/packages/module/patternfly-docs/content/extensions/data-view/examples/Functionality/Functionality.md
+sortValue: 2
+propComponents: ['DataViewToolbar', 'DataViewFilters', 'DataViewTextFilter', 'DataViewCheckboxFilter']
+sourceLink: https://github.com/patternfly/react-data-view/blob/main/packages/module/patternfly-docs/content/extensions/data-view/examples/Toolbar/Toolbar.md
 ---
 import { useMemo } from 'react';
 import { BrowserRouter, useSearchParams } from 'react-router-dom';
-import { useDataViewPagination, useDataViewSelection, useDataViewFilters, useDataViewSort } from '@patternfly/react-data-view/dist/dynamic/Hooks';
+import { useDataViewPagination, useDataViewSelection, useDataViewFilters } from '@patternfly/react-data-view/dist/dynamic/Hooks';
 import { DataView } from '@patternfly/react-data-view/dist/dynamic/DataView';
-import { BulkSelect, BulkSelectValue } from '@patternfly/react-component-groups/dist/dynamic/BulkSelect';
+import { BulkSelect, BulkSelectValue, ErrorState, ResponsiveAction, ResponsiveActions, SkeletonTableHead, SkeletonTableBody } from '@patternfly/react-component-groups';
 import { DataViewToolbar } from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
 import { DataViewTable } from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
 import { DataViewFilters } from '@patternfly/react-data-view/dist/dynamic/DataViewFilters';
 import { DataViewTextFilter } from '@patternfly/react-data-view/dist/dynamic/DataViewTextFilter';
 import { DataViewCheckboxFilter } from '@patternfly/react-data-view/dist/dynamic/DataViewCheckboxFilter';
 
-This is a list of functionality you can use to manage data displayed in the **data view**.
+The **data view toolbar** component renders a default opinionated data view toolbar above or below the data section. 
 
-# Pagination
-Allows to display data records on multiple pages and display the pagination state.
+Data view toolbar can contain pagination, bulk select, filters, actions or other child content passed. The preffered way of passing child toolbar items is using the [toolbar item](/components/toolbar#toolbar-items) component or predefined `DataViewToolbar` props for specific use cases.
+
+You can further customize toolbar interactions by referring to the additional documentation: 
+- [Actions](#toolbar-actions)
+- [Pagination](#pagination)
+- [Selection](#selection)
+- [Filters](#filters)
+
+### Toolbar example
```

Comment:
```suggestion
```
I've been suggesting to remove the "example" headings, but I'm realizing they may be necessary due to the docs framework. If so, please just disregard these suggestions and leave as is!

Reply:
Yeah, unfortunately, they are necessary